tests/performanceplatform/utils/test_collector_config.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from hamcrest import assert_that, equal_to
+from mock import patch
+from performanceplatform.utils.collector import get_config
+
+
+class ConfigTest(unittest.TestCase):
+    @patch('performanceplatform.client.collector.CollectorAPI.get_collector')
```

Comment:
That seems to be what the tests [here](https://github.com/alphagov/performanceplatform-collector/pull/126/files#diff-29c31aec230ad6f4fcf6ae117138c5b8R82) are testing?